services/src/main/resources/locale/addon/Gamification_en.properties

```
@@ -1,6 +1,12 @@
-gamification.event.title.createNewTask=Meeds - Tasks: Create task
-gamification.event.title.updateTask=Meeds - Tasks: Update task
-gamification.event.title.completeTaskCoworker=Meeds - Tasks: Task archived (coworker)
-gamification.event.title.completeTaskAssigned=Meeds - Tasks: Task archived (assignee)
-gamification.event.title.completeTask=Meeds - Tasks: Task archived
-gamification.event.title.commentTask=Meeds - Tasks: Comment task
+gamification.event.title.createNewTask=Tasks : Create task
```

Please keep existing translations

```suggestion
gamification.event.title.createNewTask=Meeds - Tasks: Create task
```